### PR TITLE
Add CODEOWNERS file to automatically assign PR reviews

### PR DESCRIPTION
**Pull request by xTVaser**
I started this as a base, please review if you don't want to be auto-added (or if you do).  Can obviously be expanded upon.

Note, even though I added myself in two spots, I won't get assigned for reviews because I do not have write access.

Documentation - https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# ACTUAL RULES
+# -------
+
+# -- NOTE ---
+# The people you choose as code owners must have write permissions for the repository. When the code owner is a team,
+# that team must have write permissions, even if all the individual members of the team already have write permissions
+# directly, through organization membership, or through another team membership.
+
+# --- GENERAL ---
+* @lightningterror
```
Comment (xTVaser) [inline .github/CODEOWNERS:52]: fyi @lightningterror incase you don't want to be tagged on every PR (unless a later rule takes precedence)

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+# --- CI/GITHUB ---
+.github/ @xtvaser
+
+# --- PLUGINS ---
+plugins/GSdx @tadanokojin
```
Comment (xTVaser) [inline .github/CODEOWNERS:58]: @tadanokojin

**Comment by lightningterror** [comment]
Do we really need it tho?

**Comment by xTVaser** [comment]
Stale / don't see this merging.  Can be re-opened if something changes.

**Pull request closed without merging**